pulp_ansible/app/tasks/collections.py

```diff
@@ -294,23 +342,30 @@ class CollectionSyncFirstStage(Stage):
     The first stage of a pulp_ansible sync pipeline.
     """
 
-    def __init__(self, remote):
+    def __init__(self, remote, repository, optimize=False):
```

Comment:
I see the release note chnaged, should this also be `optimize=True` for the default?

Reply:
it gets True from here: https://github.com/fao89/pulp_ansible/blob/7942/pulp_ansible/app/serializers.py#L101
like pulp_rpm does

Reply:
Got it! Then I guess I'm confused why it's a kwarg at all if there will always be a value specified from the serializer. Do you know why? (I don't).

Reply:
Oh actually I see it's always specified in `CollectionSyncFirstStage(remote, repository, optimize)` so this should be an args.